src/_pytest/junitxml.py

```diff
@@ -70,7 +71,7 @@ class Junit(py.xml.Namespace):
 
 
 def bin_xml_escape(arg: str) -> py.xml.raw:
-    def repl(matchobj: "re.Match[str]") -> str:
+    def repl(matchobj: "Match[str]") -> str:
```

Comment:
```suggestion
    def repl(matchobj: Match[str]) -> str:
```

Unless CI says it's necessary on Python 3.5 or such.

Reply:
let me try, I forgot you don't need to quote things imported from Typing. Thanks @bluetech